index.mjs

```diff
@@ -87,12 +150,20 @@ const stats = [...data.entries()].reduce((prev, curr) => {
 
 console.log(`Results saved to ${resultsFilePath}, total stats: ${stats}`);
 
+// exit here if save location is provided
+if (save !== false) {
+  process.exit(0);
+}
+
 const server = http.createServer(function (req, res) {
   fs.readFile(resultsFilePath, function (err, data) {
     if (err) throw err;
 
-    if (outputFormat !== 'json' && outputFormat !== 'html') {
-      const errorMessage = '❌ Output format not supported (json or html)';
+    if (!cliArguments.output.choices.includes(output)) {
```

Comment:
This doesn't work correctly.

If you use the interactive version, cliArguments.output.choices is an array of objects.
If you use the parametrized version, this is the array you expect.

Reply:
Sure thanks